features/product/managing_products/deleting_product.feature

```diff
@@ -2,7 +2,7 @@
 Feature: Deleting a product
     In order to remove test, obsolete or incorrect products
     As an Administrator
-    I want to be able to delete products from the product catalog
+    I want to be able to delete products from the product catalogue
```

Comment:
Just a thought that it will be better to use British English instead of American. We are using mostly a British one.